web/src/components/tasks/business-formation/business/NonprofitProvisions.tsx

```diff
@@ -81,7 +81,9 @@ export const NonprofitProvisions = (): ReactElement => {
     const hasError = doesFieldHaveError(fieldName);
     return (
       <WithErrorBar className="margin-top-2" hasError={hasError} type="ALWAYS">
-        <Content>{title}</Content>
+        <b>
```

Comment:
I believe we want to use `<strong>` tags over `<b>` for semantic-html reasons. https://www.pivotaltracker.com/story/show/184882929